subsys/net/lib/dns/Kconfig

```diff
@@ -20,6 +20,17 @@ config MDNS_RESOLVER
 	  This option enables multicast DNS client side support.
 	  See RFC 6762 for details.
 
+config LLMNR_RESOLVER
+	bool "Enable LLMNR support"
+	default n
+	help
+	This option enables link local multicast name resolution client side
```

Comment:
help message should be indented 2 spaces more, to follow all the other ones in this Kconfig